src/engine/Loader.ts

```diff
@@ -73,6 +74,115 @@ export class Loader extends Class implements ILoader {
     return this._imageElement;
   }
 
+  public suppressPlayButton: boolean = false;
+  protected _playButtonRootElement: HTMLElement;
+  protected _playButtonElement: HTMLButtonElement;
+  protected _styleBlock: HTMLStyleElement;
+  protected _playButtonStyles = `
+
+  /* Buttons styles start */
+
+  button#excalibur-play {
+      display: inline-block;
+      position: relative;
+      border-radius: 6px;
+      border: none;
+      /*border: 3px solid;
+      border-color: white;
+      box-shadow: 0 0 10px #ccc;*/
+      padding: 1.0rem 1.5rem 1.0rem 4rem;
+      margin: 0;
+      text-decoration: none;
+      background: #00B233;
+      color: #ffffff;
+      font-family: sans-serif;
+      font-size: 2rem;
+      line-height: 1;
+      cursor: pointer;
+      text-align: center;
+      transition: background 250ms ease-in-out, transform 150ms ease;
+      -webkit-appearance: none;
+      -moz-appearance: none;
+  }
+  
+  /*
+  button#excalibur-play {
+    display: none;
+  }*/
+  
+  button#excalibur-play:after {
+    position: absolute;
+    content: '';
+    border: 8px solid;
+    border-color: transparent transparent transparent white;
+    left: 35px;
+    top: 24px;
+    width: 0;
+    height: 0;
+  }
+  
+  button#excalibur-play:before {
+    position: absolute;
+    content: '';
+    border: 3px solid;
+    left: 19px;
+    top: 14px;
+    border-radius: 20px;
+    width: 30px;
+    height: 30px;
+    
+  }
+  
+  button#excalibur-play:hover,
+  button#excalibur-play:focus {
+      background: #00982C;
+  }
+  
+  button#excalibur-play:focus {
+      outline: 1px solid #fff;
+      outline-offset: -4px;
+  }
+  
+  button#excalibur-play:active {
+      transform: scale(0.99);
+  }
+  /* Button styles end */
+  `;
+  protected get _playButton() {
+    if (!this._playButtonRootElement) {
+      this._playButtonRootElement = document.createElement('div');
+      this._playButtonRootElement.style.position = 'absolute';
+      document.body.appendChild(this._playButtonRootElement);
+    }
+    if (!this._styleBlock) {
+      this._styleBlock = document.createElement('style');
```

Comment:
Does this work in all our supported browsers?

Reply:
@kamranayub Works in everything we support